frontend/package.json

```diff
@@ -11,10 +11,11 @@
     "@types/node": "^14.11.5",
     "@types/react": "^16.9.51",
     "@types/react-dom": "^16.9.8",
-    "antd": "^4.6.4",
+    "antd": "^4.7.0",
```

Comment:
it's prob fine but also we should make sure no other places break after we upgrade antd

Reply:
Based on my testing nowhere is broken